### ISSUE TITLE
Episodes don't contain all the providers

### ISSUE THREAD

**Issue by daniele-pintus** [lufinkey/node-justwatch-api#8]
the results of getEpisodes, getSeason and getTitle don't contain all the providers returned by the search api. 
Ex.  The search for "Vikings" returns the show that is provided by Neflix (provider id = 8) and other providers, but when i perform the getEpisodes, the result does not contain the netflix provider.

**Comment by lufinkey** [comment]
I have no control over the data that gets returned

**Comment by daniele-pintus** [comment]
Hi,

So you mean that JW itself returns incomplete data? Thank you anyway.

moreover, I found that is not possible to paginate the results (for example
the getEpisodes results), looking into your code I saw that the request
method could accept a "params" parameter that could contain the "page"
param.
By the way, the branch seem to be bugged:

// add query string if necessary

if(method==='GET'){

if(Object.keys(params) > 0) {

   reqData.path = reqData.path+'?'+QueryString.stringify(params);
}

Object.keys(params) : can never be greater than 0, it should be
Object.keys(params).length >0


*Daniele Pintus*


Il giorno gio 6 giu 2019 alle ore 14:51 Luis Finke <notifications@github.com>
ha scritto:
